DayrSuryan/DSEthiop9.xml

```diff
@@ -385,6 +456,16 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                </physDesc>
 
                <history>
+                  <origin>
+                    Patriarch <persName ref="PRS5714JohnXIX">Yoḥannǝs</persName> is mentioned in supplications, together with
+                     <persName ref="PRS7830Petros">Ṗeṭros</persName> (<locus target="#5v"/>
```

Comment:
I think we need to add nationalitiey, title/positions to the names of the hierarchs, otherwise it will be completely untransparent (now, I can only gues who is who): The Coptic partiarch John XIX (in tenure 1928-42) is mentioned together with Petros (who is Petros?) and the metropolican Qerellos of Ethiopia (in tenure 1929-50). The manuscript was likel produced in ... (preferably straight forward statements, no indirect indications inviting the reader to calculate).